functions/src/sukkirisuBot/useCase/showHelpMessage/showHelpMessage.spec.ts

```diff
@@ -25,10 +25,10 @@ describe("ShowHelpMessage", () => {
 
   it("should get correct help message", async () => {
     const actual = await useCase.getHelpMessage();
-    const expected = `メンションして以下を含めた投稿を行ってください
-スッキりす：今日のすっきりすランキングを全て表示
-個人スッキりす：ユーザとその誕生月を設定した場合、登録されているユーザのランキングを表示
-{ユーザー}は{誕生月}月生まれ：ユーザに対して誕生月を設定する。ユーザーはローマ字で指定、誕生月は数字で指定
+    const expected = `コマンド一覧。コマンドは日本語です
+Sukkirisu Bot スッキりす （全てのランキングを表示）
+Sukkirisu Bot 個人スッキりす （登録されているユーザのランキングを表示）
+Sukkirisu Bot <ユーザー名>は<誕生月>月生まれ （ユーザーを登録。ex. gansekiは12月生まれ）
```

Comment:
```suggestion
Sukkirisu Bot <ユーザー名>は<誕生月>月生まれ （ユーザーを登録。ex. abcは12月生まれ）
```

Reply:
I missed this comment, sorry.
If you don't want the message to contain your name, don't hesitate to change the message on your own.